common/app/experiments/Experiments.scala

```diff
@@ -75,3 +75,12 @@ object DCROnwardsData
       sellByDate = LocalDate.of(2023, 6, 2),
       participationGroup = Perc0C,
     )
+
+object EuropeNetworkFront
+    extends Experiment(
+      name = "europe-network-front",
+      description = "Test new europe network front",
+      owners = Seq(Owner.withGithub("rowannekabalan")),
+      sellByDate = LocalDate.of(2022, 9, 30),
```

Comment:
You might want to set this to a later date. Last I checked, expiring tests break the build and can be pesky

Reply:
Thanks for the heads up! Will push it back another month